src/components/imgix/imgixContainer/ImgixContainer.jsx

```diff
@@ -18,7 +18,15 @@ export default function ImgixContainer() {
   const [open, setOpen] = React.useState(false);
 
   function handleUrlRequest() {
```

Comment:
What are the responsibilities of this function? Can we apply any of the Clean Code principles to improve it?

Reply:
Done, I divided it on different functions and renamed the main one